modules/serverless-creating-subscriptions-ODC.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_workflows/serverless-channels.adoc
+
+[id="serverless-creating-subscriptions-ODC_{context}"]
+= Connecting a channel to a sink using subscriptions in {product-title} web console
+
+You can create a subscription to connect a service or other event sink to a channel through the {product-title} web console.
+
+.Prerequisites
+* To create a subscription through the web console, you must already have created a Knative service that can be used as an event sink.
+
+.Procedure
+. Navigate to the *Topology* page in the web console *Developer* perspective.
+. Create a subscription using one of the following methods:
+.. Click on the channel that you want to create a subscription for, and drag the arrow. The option will appear to *Add Subscription*.
++
+image::create-sub-ODC.png[Create a subscription for the channel]
++
+... Select the *Service* you want to create a subscription for from the drop-down list.
```

Comment:
Click *Add Subscription* and select...
Should this be 'Select the subscriber'?
I have not created a Knative Service and I see a note to create one, but there is a Select Subscriber drop-down, I see.
![Screenshot from 2020-10-06 19-56-02](https://user-images.githubusercontent.com/16647625/95215028-277d0080-080e-11eb-88c0-57cfc61416de.png)